wire/internal/wire/testdata/MultipleBindings/want/wire_errs.txt

```diff
@@ -0,0 +1,6 @@
+/wire_gopath/src/example.com/foo/wire.go:27:8: wire.Build has multiple bindings for example.com/foo.Foo (current binding: provider "provideFooAgain" (/wire_gopath/src/example.com/foo/foo.go:39:6); previous binding: provider "provideFoo" (/wire_gopath/src/example.com/foo/foo.go:35:6)
```

Comment:
Do you think it's too fragile to include the line numbers here?
I don't think we've done that before, but they are interspersed with the actual error text here and so hard to avoid.
Just verifying "multiple bindings" isn't useful for this test.

Reply:
I've tried to avoid adding line numbers just because it makes it harder to update the tests, but I don't have a suggestion as to how to make this better without changing the test infrastructure to permit regexes or some such. (Might not be the worst idea.)

Reply:
Filed an issue for supporting regexes.